client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -127,8 +132,10 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                       lb-posts-list
                       lb-posts-blog-id="blog._id"
                       lb-posts-status="open"
-                      lb-posts-allow-unpublish="true"
-                      lb-posts-allow-reordering="true"
+                      lb-posts-allow-unpublishing="blogSecurityService.canPublishAPost"
```

Comment:
I still don't understand what are the advantages.
Either we do the mapping into a new directive to link privileges to different types of post list, or we do it in the directive parameters, as I did.
I really prefer the second option, since this is more explicit which actions are allowed where you use the directive. Also it make the directive usable for other purposes without having to modify a service.